task_sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -429,9 +429,7 @@ def run(ti: RuntimeTaskInstance, log: Logger):
     except SystemExit:
         ...
     except BaseException:
-        # TODO: Handle TI handle failure
```

Comment:
We still need this comment, as it involves running callbacks etc.

Reply:
Replaced with:
```
# TODO: Run task failure callbacks here
```